src/main/java/adris/altoclef/commands/FullBrightCommand.java

```diff
@@ -0,0 +1,36 @@
+package adris.altoclef.commands;
+
+import net.minecraft.client.MinecraftClient;
+
+import adris.altoclef.AltoClef;
+import adris.altoclef.Debug;
+import adris.altoclef.commandsystem.Arg;
+import adris.altoclef.commandsystem.ArgBase;
+import adris.altoclef.commandsystem.ArgParser;
+import adris.altoclef.commandsystem.Command;
+import adris.altoclef.commandsystem.CommandException;
+
+public class FullBrightCommand extends Command
+{
+
+    public FullBrightCommand() throws CommandException
+    {
+        super("fullBright", "sets gamma really high", new Arg(String.class, "multiplier", "1", 0 ));
```

Comment:
Instead of String.class I highly recommend using Float.class or Double.class, with the default value at 1.0 or something like that

Maybe a personal decision, but I'd do one of two things.
1) Rename this to "gamma" and have the float argument set the gamma directly
2) Keep this as fullbright but remove the multiplier value, as it might just make the command more confusing. If  you really want to keep it in or see it as useful, disregard this.

Reply:
Ok yeah i you're right, i will make it so it just sets the gamma, it's much easyer making it that way.